charts/kubescape-cloud-operator/tests/snapshot_test.yaml

```diff
@@ -0,0 +1,19 @@
+suite: Full Snapshot
+tests:
+  - it: matches the snapshot
+    asserts:
+      - matchSnapshot: {}
+    capabilities:
+      apiVersions:
+        - batch/v1/CronJob
+    set:
+      account: 9e6c0c2c-6bd0-4919-815b-55030de7c9a0
```

Comment:
I hope this is not a real account :)

Reply:
it was our 1-4-3 account, I don't think it's critical?